docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -164,6 +168,17 @@ connected agents?](#how-can-i-find-version-information-on-connected-agents).
 For more information about automatic updates and compatibility issues, contact
 [Teleport support](https://goteleport.com/support/).
 
+### Why wasn't my Teleport Enterprise Cloud instance upgraded to the latest version?
+
```

Comment:
Here's my suggestion, take it or leave it.

>>> The Teleport Control plane supports agents that are on the same major version or one major version behind. If your agents are not up to date, we will withhold major version upgrades in order to break connectivity to your resources. For example, if your control plane is currently running Teleport 15 and you have agents running Teleport 14, we will not be able to upgrade your control plane to Teleport 16 until all agents are running Teleport 15.

Reply:
thanks @zmb3 , please see update